waimak-core/src/main/scala/com/coxautodata/waimak/dataflow/spark/ParquetDataCommitter.scala

```diff
@@ -27,49 +29,53 @@ import scala.util.{Failure, Success, Try}
   * @param toRemove   optional function that takes the list of available snapshots and returns list of snapshots to remove
   * @param conn       optional connector to the DB.
   */
-class ParquetDataCommitter(val baseFolder: String
-                           , val snapFolder: Option[String]
-                           , val toRemove: Option[CleanUpStrategy[FileStatus]]
-                           , val conn: Option[HadoopDBConnector])
-      extends DataCommitter with Logging {
+class ParquetDataCommitter(private val baseFolder: String
```

Comment:
Maybe make these not vals (just constructor params) if we don't want to expose them